vta/python/vta/environment.py

```diff
@@ -241,7 +243,7 @@ def target_host(self):
             return "llvm -target=armv7-none-linux-gnueabihf"
         if self.TARGET == "ultra96":
             return "llvm -target=aarch64-linux-gnu"
-        if self.TARGET in ["sim", "tsim"]:
+        if self.TARGET in ["sim", "tsim", "intelfocl"]:
```

Comment:
We may want to be a little more specific about the hardware target

Reply:
ok. sure. we'll change the naming

Reply:
Could you follow up and rename to say, "arria10"?